package.json

```diff
@@ -11,14 +11,14 @@
     "build:ssr": "npm run build && npm run webpack:server",
     "build:prerender": "npm run build:ssr && npm run generate:prerender",
     "build:prerender-ghpages": "npm run build:ghpages && npm run webpack:server && npm run generate:prerender",
-    "build:app": "ng build --prod",
-    "build:server": "ng build --prod --app 1 --output-hashing=false",
-    "build:lib": "npm run clean && npm run packagr && npm run build:lib:sass && npm run copy:styles && npm run copy:readme",
-    "build:lib:sass": "node-sass src/lib/scss/ngx-drag-to-select.scss -o dist/lib",
-    "packagr": "ng-packagr -p src/lib/package.json",
+    "build:app": "ng build --prod --project=app",
+    "build:server": "ng build --prod --project=server",
+    "build:lib": "npm run clean && npm run build:lib:code && npm run build:lib:sass && npm run copy:styles && npm run copy:readme",
+    "build:lib:code": "ng build --prod --project=ngx-drag-to-select",
+    "build:lib:sass": "node-sass projects/ngx-drag-to-select/src/scss/ngx-drag-to-select.scss -o dist/ngx-drag-to-select",
     "deploy": "ngh --dir dist/browser",
```

Comment:
Is this used @d3lm ?

Reply:
This is only meant to be used locally, if needed.